srcpkgs/python3-jeepney/template

```diff
@@ -1,17 +1,23 @@
 # Template file for 'python3-jeepney'
 pkgname=python3-jeepney
-version=0.4.3
-revision=2
+version=0.6.0
+revision=1
 wrksrc="jeepney-${version}"
 build_style=python3-module
 hostmakedepends="python3 python3-setuptools"
 depends="python3"
+checkdepends="python3-pytest python3-testpath"
 short_desc="Low-level, pure Python DBus protocol wrapper"
 maintainer="Orhpaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="https://gitlab.com/takluyver/jeepney"
+changelog="https://gitlab.com/takluyver/jeepney/-/raw/${version%.*}/docs/release-notes.rst"
```

Comment:
Please use version from master.

Reply:
:) Just say master if you find more, I forgot which all I changed them on. Save you some typing :D